tests/containers/buildah_integration.pm

```diff
@@ -76,12 +75,12 @@ sub run {
 
     switch_to_user;
 
-    assert_script_run "cd $test_dir";
-
     # Download buildah sources
-    $buildah_version = script_output "buildah --version | awk '{ print \$3 }'";
-    script_retry("curl -sL https://github.com/containers/buildah/archive/refs/tags/v$buildah_version.tar.gz | tar -zxf -", retry => 5, delay => 60, timeout => 300);
-    assert_script_run "cd $test_dir/buildah-$buildah_version/";
+    my $buildah_version = script_output "buildah --version | awk '{ print \$3 }'";
+    my $url = get_var("BUILDAH_BATS_URL", "https://github.com/containers/buildah/archive/refs/tags/v$buildah_version.tar.gz");
+    assert_script_run "mkdir -p $test_dir";
+    script_retry("curl -sL $url | tar -zxf - --strip-components 1 -C $test_dir", retry => 5, delay => 60, timeout => 300);
+    assert_script_run "cd $test_dir";
```

Comment:
Nitpick: Move this up, that saves you the `-C` in the tar command.

Reply:
Fixed.